### PR TITLE
exclude modes for csv configurations

### LINKED ISSUE #2901: Exclude modes in CSV configurations

### PR DESCRIPTION
**Pull request by rajnikantsh**
Resolves #2901 

This change is [Reviewable](https://reviewable.io/reviews/lbnl-ucb-sti/beam/2902)

### CODE REVIEW

**Comment by rajnikantsh** [comment]
test!

File: src/main/scala/beam/utils/scenario/urbansim/censusblock/merger/PersonMerger.scala [hunk 1]
```diff
@@ -16,6 +16,7 @@ class PersonMerger(inputHousehold: Map[String, InputHousehold]) extends Merger[I
       HouseholdId(inputPersonInfo.householdId),
       0,
       inputPersonInfo.age,
+      "",
```
Comment (carloscaldas) [inline src/main/scala/beam/utils/scenario/urbansim/censusblock/merger/PersonMerger.scala:19]: use parameter names here

Comment (rajnikantsh) [reply src/main/scala/beam/utils/scenario/urbansim/censusblock/merger/PersonMerger.scala]: use parameters name

**Comment by rajnikantsh** [comment]
test!

**Comment by rajnikantsh** [comment]
test!

**Comment by carloscaldas** [comment]
Test!

**Comment by rajnikantsh** [comment]
test now!

**Pull request merged**